### ISSUE TITLE
Failed to resolve: use of undeclared type "OutputType"

### ISSUE THREAD

**Issue by AllenDang** [fltk-rs/fl2rust#4]
Here is the fluid file
```
Fl_Output output {
  xywh {5 45 570 370} type Multiline
}
```

It generates like this
```rust
let mut output = Output::new(5, 45, 570, 370, None);
output.set_type(OutputType::Multiline);

```

Error message is 
```
undeclared type "OutputType"
```

**Comment by MoAlyousef** [comment]
Hi
I just published a fix in 0.4.6

**Comment by AllenDang** [comment]
@MoAlyousef Thanks! It is fixed.
